source/documentation/ways-of-working/index.html.md.erb

```diff
@@ -6,3 +6,206 @@ review_in: 3 months
+- **Innovation:** Embrace new ideas and technologies that help us improve our service offering.
+- **Transparency:** Maintain clear visibility into our processes, decisions, and progress.
+
+## Team Roles
+
+- **Team Members:** Comprising a mix of developers, data and platform engineers,
+responsible for the delivery of tasks and active participation in all Agile ceremonies.
+- **Product Manager (PM):** Oversees the product backlog,
+ensuring it aligns with stakeholder needs and business objectives.
+- **Delivery Manager (DM):** Facilitates the Agile process,
+removing blockers and ensuring timelines are met.
+
+## Communication and Delivery
+
+- **Daily [Stand-ups](#stand-up):** A quick get-together to share progress updates.
+- **Ceremonies:** How we make sure our team stays aligned as we work to deliver during Agile Sprints.
+
+## Ceremonies
+
+### Schedule
+
+This team uses **Fortnightly Sprints** This is our timeframe for planning, executing, and reviewing work.
+
+The team has the following ceremonies:
+
+| Ceremony                                      | Occurence   | Day             | Time          |
+| --------------------------------------------- | ----------- | --------------- | ------------- |
+| [Stand Up](#stand-up)                         | Daily       | Monday - Friday | 10:45 - 11:00 |
+| [Show & Tell](#show-and-tell)                 | Fortnightly | Monday          | 15:00 - 15:30 |
+| [Roadmap Monthly Review](#roadmap-review)     | Fortnightly | Monday          | 15:30 - 16:15 |
+| [Refinement 1](#refinement)                   | Fortnightly | Monday          | 13:00 - 14:00 |
+| [Peer Review](#peer-review)                   | Fortnightly | Thursday        | 15:30 - 16:30 |
+| [Refinement 2](#refinement)                   | Fortnightly | Thursday        | 12:00 - 12:45 |
+| [Sprint Retrospective](#sprint-retrospective) | Fortnightly | Thursday        | 11:00 - 12:00 |
+| [Sprint Planning](#sprint-planning)           | Fortnightly | Thursday        | 13:00 - 14:00 |
+
+### Stand Up
+
+Our stand-ups run daily at 10:45 and last for fifteen minutes.
+We use Google Meet and follow one of two formats:
+
+- **Walk The Board**
+
+  - Starting with the “Done” column we discuss any active issues on our
+  [GitHub Project Board](https://github.com/orgs/ministryofjustice/projects/57/views/5).
+
+- **Yesterday, Today, Blocked**
+
+  - A round robin of each team member discussing their issues and any blockers.
+
+Standup is run by either the PM, DM, or whoever wants to run it.
+
+Ad-hoc discussions may spin out from stand-up.
+These should take place after the meeting ends if time allows,
+or should be arranged for the next available time.
+
+### Show and Tell
+
+- Near the beginning of each sprint we have a thirty-minute show and tell showcase that provides an opportunity to share our progress from the previous sprint with the wider organisation.
+- [TODO: show and tell content]
```

Comment:
what's this TODO for?

Reply:
I wasn't sure if there was a consistent 'content' / order of operations for the Show and Tell. If there is, someone should put it here

Reply:
Ah I don't know... @seanprivett?